commit cdebfd27292ecdebe7d493830354e302368b3188
Author: Joel Stanley <joel@jms.id.au>
Date:   Wed Jul 5 12:00:56 2023 +0930

    powerpc/config: Disable SLAB_DEBUG_ON in skiroot
    
    In 5.10 commit 5e84dd547bce ("powerpc/configs/skiroot: Enable some more
    hardening options") set SLUB_DEBUG_ON.
    
    When 5.14 came around, commit 792702911f58 ("slub: force on
    no_hash_pointers when slub_debug is enabled") print all the
    pointers when SLUB_DEBUG_ON is set. This was fine, but in 5.12 commit
    5ead723a20e0 ("lib/vsprintf: no_hash_pointers prints all addresses as
    unhashed") added the warning at boot.
    
    Disable SLAB_DEBUG_ON as we don't want the nasty warning. We have
    CONFIG_EXPERT so SLAB_DEBUG is enabled. We do lose the settings in
    DEBUG_DEFAULT_FLAGS, but it's not clear that these should have been
    always-on anyway.
    
    Signed-off-by: Joel Stanley <joel@jms.id.au>
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
    Link: https://msgid.link/20230705023056.16273-1-joel@jms.id.au

diff --git a/arch/powerpc/configs/skiroot_defconfig b/arch/powerpc/configs/skiroot_defconfig
index 71cfb990a74f..8d3eacb50d56 100644
--- a/arch/powerpc/configs/skiroot_defconfig
+++ b/arch/powerpc/configs/skiroot_defconfig
@@ -289,7 +289,6 @@ CONFIG_LIBCRC32C=y
 # CONFIG_XZ_DEC_SPARC is not set
 CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
-CONFIG_SLUB_DEBUG_ON=y
 CONFIG_SCHED_STACK_END_CHECK=y
 CONFIG_DEBUG_STACKOVERFLOW=y
 CONFIG_PANIC_ON_OOPS=y